charts/argo-cd/templates/argocd-aggregate-roles.yaml

```
@@ -0,0 +1,67 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "argo-cd.fullname" . }}-aggregate-to-view
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.controller.name "name" .Values.controller.name) | nindent 4 }}
```

I think we do not need `component` and `name` labels set for these resources.
```suggestion
    {{- include "argo-cd.labels" (dict "context" .) | nindent 4 }}
```

---

I was not sure how people are using existing labels and therefore decided to align on the existing ones. I'm fine with the removal